arduino/builder/testdata/TestLoadSketchFolder/header.h

```diff
@@ -0,0 +1 @@
+#define TRUE FALSE
```

Comment:
Do you mind adding a comment here? Seeing such a declaration looks scary even if in the test folder.

Reply:
Not sure if actually needed, I changed it into something else more obvious